### PR TITLE
Adding PerfScale jobs to multus

### PR DESCRIPTION
**Pull request by jtaleric**
Adding some perfscale jobs to multus

Each of these jobs will be `optional:true` and  `always_run:false`

### CODE REVIEW

**Comment by jluhrsen** [comment]
/pj-rehearse

let's just see how these look in rehearsals as a double check

also, I'd update the commit message to indicate that you are going `optional:true` `always_run:false`

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-multus-cni-master-qe-perfscale-aws-ovn-small-cluster-density

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-multus-cni-master-qe-perfscale-aws-ovn-small-cluster-density

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-multus-cni-master-qe-perfscale-aws-ovn-small-cluster-density

File: ci-operator/config/openshift/multus-cni/openshift-multus-cni-master.yaml [hunk 1]
```diff
@@ -51,6 +51,72 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-upgrade-aws
+- always_run: false
+  as: qe-perfscale-aws-ovn-medium-cluster-density
+  optional: true
+  steps:
+    cluster_profile: aws-perfscale
```
[the remark this answers is not in the document]
Comment (jtaleric) [reply ci-operator/config/openshift/multus-cni/openshift-multus-cni-master.yaml:58]: What is the "default" cluster profiles?

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-multus-cni-master-qe-perfscale-aws-ovn-small-cluster-density

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-multus-cni-master-qe-perfscale-aws-ovn-small-cluster-density

**Comment by jtaleric** [comment]
/pj-rehearse pull-ci-openshift-multus-cni-master-qe-perfscale-aws-ovn-small-cluster-density

**Review by s1061123 — APPROVED** [review]

**Pull request merged**